src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/nonemptyatclausedescription/InputNonEmptyAtclauseDescription.java

```diff
@@ -78,7 +88,7 @@ public int foo2(String a, int b, double c) throws Exception
          * @deprecated
          * @throws Exception
          * @deprecated
-         */
+         */ // violation above
```

Comment:
Problem here. Violation is on line 90 but if we add text, it is no longer a violation.

Reply:
https://github.com/checkstyle/checkstyle/issues/10378